### PR TITLE
Unit Test

### PR DESCRIPTION
**Pull request by NorakGithub**
- [x] Common Best Practices
- [x] Fixtures
- [x] Parallel Testing

### CODE REVIEW

**Review by powerdefy — APPROVED** [review]
look good

File: django/project-structure/README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ The difference in structure of each project makes it developers difficult to und
     │   │   ├── common/ <sup><a href="#sup19sup-codecommoncode-application">19</a></sup>
     │   │   │   └── ..
     │   │   ├── [app1_name]/ <sup><a href="#sup1sup-application-name">1</a></sup>
+    │   │   │   ├── fixtures/ <sup><a href="#sup21sup-fixtures">21</a></sup>
```
Comment (earthpyy) [inline django/project-structure/README.md:21]: Thanks krubbbb

File: django/unit-test/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+```shell script
+$ python manage.py dumpdata app.Model >> path/app/fixtures/model.json
+```
+
+If you have a set of fixtures can be reusable, consider creating a reusable constants.
+
+```python
+# Yes
+## fixture_templates.py
```
Comment (earthpyy) [inline django/unit-test/README.md:134]: Do you have any suggestions on where should this file in krub?

File: django/unit-test/README.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+class PersonTest(tests.TestCase):
+    fixtures = [
+        'users.json',
+        'people.json',
+    ]
+```
+
+## Parallel Testing
+
+Running test in parallel could speed up your test suite. [Read More](https://docs.djangoproject.com/en/3.1/ref/django-admin/#cmdoption-test-parallel)
```
Comment (earthpyy) [inline django/unit-test/README.md:168]: Maybe the sample usage krubbb?

```shell script
docker-compose exec django python manage.py test --parallel
```

**Review by earthpyy — CHANGES_REQUESTED** [review]

File: django/unit-test/README.md (hunk 2 above)
Comment (NorakGithub) [reply django/unit-test/README.md:134]: I think should be in common

File: django/unit-test/README.md (hunk 3 above)
Comment (NorakGithub) [reply django/unit-test/README.md:168]: Okay krub

**Review by earthpyy — APPROVED** [review]
Cool krubb

**Pull request merged**